scripts/biomedical/NCBI_Taxonomy/schema_mfc/ncbi_taxonomy_schema.mcf

```diff
@@ -0,0 +1,81 @@
+Node: dcid:Taxon
```

Comment:
It's not necessary to include this new schema as that's auto pushed to github at a different location - in my understanding. But I think it's a good idea to include the output of the auto generated schema "ncbi_taxonomy_schema_enum.mcf" file

Reply:
I've added a sample file named ncbi_taxonomy_schema_enum.mcf and updated the README to reflect this change.